### DISCUSSION TITLE
Enhancement: Please bring back instrumentals on the freeplay song selection

### DISCUSSION THREAD

**Discussion by lauyc-c** [FunkinCrew/Funkin#1663] [Ideas]
## What is your suggestion, and why should it be implemented?
Before the Week 7 update, every time I am selecting for a song in freeplay, the instrumentals of the song will play in place of the usual menu theme, but it's gone after the latest update. It's currently the only way we can listento just the instrumentals of the songs inside the game, and some other rhythm games do that too while selecting for the songs. I dunno if it's a bug or anything but I'd appreciate it if you could bring it back.

**Comment by haya3218** [comment]
Its only in html5 compiles.
This is not an issue in a desktop version
